package.json

```diff
@@ -17,9 +17,9 @@
         "test:mini-rx-store-ng": "nx test mini-rx-store-ng",
         "test:mini-rx-store-ng:watch": "nx test mini-rx-store-ng --watch",
         "test:mini-rx-store-ng:coverage": "nx test mini-rx-store-ng --codeCoverage",
-        "deploy:docs": "npm run deploy --prefix docs",
+        "deploy:docs": "cd docs && yarn deploy && cd ..",
```

Comment:
The reason I added it is that if you would run this as part of a pipeline operation, it's probably best to navigate back so the next commands are executed from the expected cwd?

Reply:
I am not sure if it would be needed in a CI pipeline

The docs will not be part of CI pipeline anytime soon...

So I would rather say we add the `cd ..` part when we really need it.

Reply:
```suggestion
        "deploy:docs": "cd docs && yarn deploy"
```

Reply:
My suggestion was not OK. Comma is still needed. 🤦‍♂️

Reply:
No worries, also missed that one :)